addons/mail/static/src/core/common/store_service.js

```diff
@@ -203,6 +203,17 @@ export class Store extends BaseStore {
         },
     });
 
+    standaloneInboxMessages = Record.many("mail.message", {
+        compute() {
+            const tab = this.store.discuss.activeTab;
+            if (tab !== "main") {
+                return [];
+            }
+            const messages = this.store.inbox.messages.filter(m => !m.thread);
+            return messages.sort((m1, m2) => m2.datetime - m1.datetime);
```

Comment:
Theoretically 2 messages could have same `datetime` and order is unclear there.

Since this uses `messages` that is ordered from oldest to newest, if 2 messages have same `datetime` then the oldest id comes first, which is not correct with messaging menu that must show more recent items first.

I'd order by id if the datetime are the same.